sdk/tests/utilities/test_token_refresh.py

```diff
@@ -97,40 +90,30 @@ def test_get_token_with_proxy_from_config(self):
         if secrets["proxy"].get("address", None) is None:
             self.skipTest(f"missing proxy configuration")
 
-        secrets_file = TempFileManager.create_temp_file(secrets)
+        TempFileManager.create_temp_file(secrets)
 
-        original_token, refresh_token = tu.get_okta_tokens(secrets_file.name)
+        refreshed_token = RefreshingToken(api_configuration=self.config,
+                                          expiry_offset=3599)
 
-        proxy_config = ProxyConfig(
-            address=secrets["proxy"]["address"],
-            username=secrets["proxy"]["username"],
-            password=secrets["proxy"]["password"]
-        )
+        self.assertIsNotNone(refreshed_token)
 
-        proxies = proxy_config.format_proxy_schema()
```

Comment:
Not required as the RefreshingToken now picks this up from the provided self.config.